### ISSUE TITLE
AttributeError: module 'fpdf' has no attribute 'svg'

### ISSUE THREAD

**Issue by muety** [PyFPDF/fpdf2#316]
I'm using `fpdf2==2.4.6`. Trying to follow [this guide](https://pyfpdf.github.io/fpdf2/SVG.html) to include an SVG image, I get the above error message.

```python
import fpdf
svg = fpdf.svg.SVGObject.from_file("/tmp/image.svg")
```

**Comment by Lucas-C** [comment]
This feature has been merged just yesterday and has not been released yet.

With @torque we would be very happy to have feedback from users though!

To test the SVG feature, you can install the latest version of `fpdf2` from the `master` branch of this git repo:

    pip install git+https://github.com/PyPDF/fpdf2.git@master

**Comment by muety** [comment]
What a coincidence! Thanks a lot, works now.

**Comment by francescoalongi** [comment]
Hello,
first of all thank you for this work, I was messing around with the old fpdf package and then you saved my life :)

Do you already know when the version with svg feature will be released? I really need it as a release!

Thanks.

**Comment by Lucas-C** [comment]
@francescoalongi : probably before the end of the month.

Have you tried the SVG features? If so, what is your feedback? On its usage, its documentation, any difficulty you had?

**Comment by Lucas-C** [comment]
This is as now been released as part of `v2.5.0`
